test/Makefile

```diff
@@ -26,10 +26,19 @@ CXXFLAGS += -g -Wall -Wextra -pthread -std=c++11 -fvisibility=hidden
 
 # All tests produced by this Makefile. Remember to add new tests you
 # created to the list.
-TESTS = indexer hdr
+TESTS = indexer        \
+        hdr            \
+        display_matrix
 
+TESTS = display_matrix
```

Comment:
You added this line by mistake, and nuked the rest of the tests.

Reply:
whoops yes i was skipping the other ones, updating...